scripts/docker_release/build_algod_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+ALGOD_INSTALL_TAR_FILE=$1
+if [[ $ALGOD_INSTALL_TAR_FILE == "" ]]
+then
+   echo "specify filepath of base install file"
+   exit 1
+fi
+
+echo "building 'Dockerfile_algod_release' with install file $ALGOD_INSTALL_TAR_FILE"
+cp $ALGOD_INSTALL_TAR_FILE ./temp_install.tar.gz
+docker build --build-arg ALGOD_INSTALL_TAR_FILE=temp_install.tar.gz . -t algorand/release -f Dockerfile_algod_release
```

Comment:
Wronge filename here: Dockerfile_algod_release should be Dockerfile-algod-release

Reply:
Fixed that, testing was done using a mirror on an EC2 instance, so missed this when committing files.